options.go

```diff
@@ -41,6 +41,15 @@ func CustomizeRequest(src GenericContainerRequest) CustomizeRequestOption {
 	}
 }
 
+// BuildFromDockerfile allows to build a container from a Dockerfile
+func BuildFromDockerfile(ctx context.Context, df FromDockerfile) CustomizeRequestOption {
```

Comment:
suggestion: remove ctx as it's not used.

Reply:
I thought I had removed it 🤦